plugins/modules/aci_ip_sla_monitoring_policy.py

```diff
@@ -0,0 +1,388 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Tim Cragg (@timcragg)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_ip_sla_monitoring_policy
+short_description: Manage IP SLA Monitoring Policies (fv:IPSLAMonitoringPol)
+description:
+- Manage Internet Protocol (IP) Service Level Agreement (SLA) Monitoring Policies used for L4-L7 Policy Based Redirection
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  name:
+    description:
+    - The SLA Policy name.
+    type: str
+    aliases: [ sla_policy ]
+  description:
+    description:
+    - The SLA Policy description.
+    type: str
+  sla_type:
+    description:
+    - The type of monitoring.
+    - The APIC defaults to C(icmp) when unset during creation.
+    type: str
+    choices: [ icmp, tcp, l2ping, http ]
+  sla_port:
+    description:
+    - The Port to monitor for TCP SLAs.
```

Comment:
should we add the ranges allowed for each input, same applies for all the other inputs